app/views/foreman_leapp/job_templates/leapp_preupgrade.erb

```diff
@@ -13,15 +13,22 @@ template_inputs:
   required: true
   default: "ga"
   options: "ga\ne4s\neus\naus"
+- name: Target Version
+  description: "Specify the target release version of the operating system to upgrade to. By default, the latest release is used."
+  input_type: user
+  required: false
 %>
 <%
-  channel_opts = input('Channel') == 'ga' ? '' : "--channel #{input('Channel')}"
+  options = []
+  options << ['channel', input('Channel')] if input('Channel') != 'ga'
+  options << ['target', input('Target Version')] if input('Target Version').present?
```

Comment:
In the original PR the target options were placed before channel ones. Just to be sure: does it matter?

Reply:
It doesn't seem to matter